tests/Subscriber/FrontendSubscriberTest.php

```diff
@@ -54,6 +83,15 @@ public function testHeaderPageletLoadedEvent(): void
             ->disableOriginalConstructor()
             ->getMock();
 
+        $request = new Request(
+            $requestParams[0],
+            $requestParams[1],
+            $requestParams[2],
+            $requestParams[3],
+            $requestParams[4],
+            $requestParams[5]
+        );
```

Comment:
```suggestion
        $request = new Request(... $requestParams);
```

Reply:
@TobiasGraml11 thank you for this syntax :)